examples/somewhat-realistic/flake.nix

```diff
@@ -73,38 +84,14 @@
         ];
       };
 
-
-
-
-
```

Comment:
Yeah, for this particular example I added all this extra spacing so it would be easier to read for newcomers, not sure if it should have been left as it is or not :/